test/common/cdp.py

```diff
@@ -197,7 +201,10 @@ def browser_cmd(self, cdp_port, env):
             with open(os.path.join(profile, "handlers.json"), "w") as f:
                 f.write('{"defaultHandlersVersion":{"en-US":4},"mimeTypes":{"application/xz":{"action":0,"extensions":["xz"]}}}')
 
-            cmd = [exe, "-P", "blank", "--window-size=1920,1200", "--remote-debugging-port=%i" % cdp_port, "--no-remote", "localhost"]
+            ws = "1920,1200"
+            if self.mobile:
+                ws = "414,736"  # iPhone 6/7/8 Plus
```

Comment:
Please don't repeat the numbers -- can you set self.width and self.height in the constructor and just assemble the strings here? (comma vs 'x')